### PR TITLE
Feature/trailer

### PR DESCRIPTION
**Pull request by Mariusz-web**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,9 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Movie Explorer</title>
     <link rel="stylesheet" href="styles/styles.scss">
+    
+    <link href="//vjs.zencdn.net/7.10.2/video-js.min.css" rel="stylesheet">
+    <script src="//vjs.zencdn.net/7.10.2/video.min.js"></script>
```
Comment (Veryyapeee) [inline index.html]: You installed the Video.js via NPM, why are you linking the CDN in here?

File: src/components/Trailer/Trailer.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+
+export const TrailerMovie = ( videoUrl ) => {
+    const template =  `
+        <video
+        id="my-video"
+        class="video-js vjs-default-skin vjs-big-play-centered"
+        controls
+        preload="auto"
+        width="640"
```
Comment (Veryyapeee) [inline src/components/Trailer/Trailer.js]: This should be responsive I think

File: src/components/Trailer/Trailer.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        height="268"
+        poster="MY_VIDEO_POSTER.jpg"
```
Comment (Veryyapeee) [inline src/components/Trailer/Trailer.js]: What is that? :o

File: src/screens/Details.js [hunk 4]
```diff
@@ -1,10 +1,11 @@
 import { AssetDetailsDescription } from '../components/AssetDetails/Description.js';
 import { render } from '../shared/dom.js';
 import ApiProvider from '../providers/ApiProvider';
+import { TrailerMovie } from '../components/Trailer/trailer.js';
 
 const template = `
 <div id="details-screen" class="details">
-    <template class="details__trailer"></template>
+    <template class="details__trailer">${TrailerMovie}</template>
```
Comment (Veryyapeee) [inline src/screens/Details.js]: You have to actually pass an argument in here `TrailerMovie(url)`

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/screens/Details.js [hunk 5]
```diff
@@ -1,10 +1,11 @@
 import { AssetDetailsDescription } from '../components/AssetDetails/Description.js';
 import { render } from '../shared/dom.js';
 import ApiProvider from '../providers/ApiProvider';
+import { TrailerMovie } from '../components/Trailer/trailer.js';
```
Comment (Veryyapeee) [inline src/screens/Details.js]: Wrong name in import, that's why pipeline failed

File: index.html (hunk 1 above)
Comment (Mariusz-web) [reply index.html]: I try something and forgot about it

File: src/components/Trailer/Trailer.js (hunk 2 above)
Comment (Mariusz-web) [reply src/components/Trailer/Trailer.js]: i change to width="100%" and height="auto"

File: src/components/Trailer/Trailer.js (hunk 3 above)
Comment (Mariusz-web) [reply src/components/Trailer/Trailer.js]: This is poster before turning on the video

**Comment by Veryyapeee** [comment]
Pipeline still not fixed

**Review by Veryyapeee — APPROVED** [review]
I see you gave up on video.js, it's fine. it's working :D

File: src/screens/Details.js [hunk 6]
```diff
@@ -17,7 +18,12 @@ export const DetailsScreen = async ({ renderOn }) => {
     const searchID = params.get('id');
     const movie = await ApiProvider.getTitleDetails(searchID);
 
-    render({ on: renderOn, html: template });
+    let trailer = '';
+    if (movie.trailer && movie.trailer.linkEmbed) {
+        trailer = `<iframe class="iframe" src="${movie.trailer.linkEmbed}"></iframe>`;
```
Comment (Veryyapeee) [inline src/screens/Details.js]: Change the class to "video" for example, would be more accurate imo

**Review by Veryyapeee — APPROVED** [review]

**Review by KonTra7cf — APPROVED** [review]

**Pull request merged**